README.md

```diff
@@ -98,19 +116,76 @@ The instructions below mention **Git Bash** and **Windows command prompt**, you
 
 [Follow the instructions](https://github.com/git-guides/install-git#install-git-on-windows).
 
+**Setup Git Credential Manager Core (Windows Users Only)**
+
+*Note: Proceed with this step only after installing Git*
+
+Git Credential Manager (GCM) Core enables you to authenticate a remote Git server, even if you have a complex authentication pattern like two-factor authentication, or using SSH remote URLs that require an SSH key password for every Git push.
+
+- Open up Windows Terminal
+- Enter this command and run: `git config --global credential.helper "/mnt/c/Program\ Files/Git/mingw64/libexec/git-core/git-credential-manager-core.exe"`
+
 ### Mac installation
 
 [Follow the instructions](https://github.com/git-guides/install-git#install-git-on-mac).
 
 ---
 
+## Setup SSH Keys for GitHub
+
+- Open Terminal
+
+- Run the following code, substituting in your GitHub email address in the last placeholder: `ssh-keygen -t ed25519 -C "your_github_email@email.com"`
+
+- When you're prompted to "Enter a file in which to save the key," press Enter to save the file in the default location. Note the default location as shown in the image below:
+![defaultLocation](sshSetup.png)
+
+- When prompted, type a secure passphrase. Note down the passphrase for use later.
+
+- Once that is successful, run `cat /Users/you/.ssh/id_ed25519.pub`. Replace `/Users/you/.ssh/id_ed25519` with the default location shown earlier in step 3. Take note of the output.
+
+- Navigate to your GitHub profile and go to settings. Click on the Tab named "SSH and GPG keys"> Click 'New SSH Key'
+
+- In the box labelled Key, input the output you got from step 5 and then input a title. This could be just the device you are working on .e.g 'My Silver Mac'.
+
+- Finally, click 'Add SSH Key'
+
+### Configure Default Branch for Git Init
+
+- Verify your version of Git by running `git --version`
+If your version number is below 2.28, then proceed with the steps below. Otherwise, skip to the steps under Version 2.28.
+
+#### Below Version 2.28
+- Open the Terminal
+- Download the configure script by running: `wget https://raw.githubusercontent.com/blackcodherbootcamp/setup-guide/main/script.sh`
+- Then run `./script.sh`

```

Comment:
How do they run this script?

Reply:
They just have to put the command on line 161 and hit enter.

Reply:
Line 160 gets them to download the file and then line 161 executes it